src/claim_extractor/llm_extract.py

```diff
@@ -3,63 +3,187 @@
 import logging
 from typing import List, Dict, Any, Optional
 import os
-from langchain_core.prompts import ChatPromptTemplate, SystemMessagePromptTemplate, HumanMessagePromptTemplate
+from langchain.prompts import ChatPromptTemplate, SystemMessagePromptTemplate, HumanMessagePromptTemplate
 from langchain_anthropic import ChatAnthropic
-from langchain_core.language_models.base import BaseLanguageModel
+from langchain.base_language import BaseLanguageModel
 from .schemas.loader import load_schema_info, LINKED_TRUST
 
 def default_llm():
     return ChatAnthropic(
-        model=os.getenv("CLAUDE_MODEL", "claude-3-5-sonnet-20241022"),
-        temperature=0,
-        max_tokens=int(os.getenv("CLAUDE_MAX_TOKENS", 4096)))
+        model=os.getenv("CLAUDE_MODEL"),  # This is the current Sonnet model
+        temperature=0,  # 0 to 1, lower means more deterministic
+        max_tokens=os.getenv("CLAUDE_MAX_TOKENS", 4096))
```

Comment:
please remove this change, its ok to have a default and cleanup of string, please revert this change.